### PR TITLE
Stars were created

### PR DESCRIPTION
**Pull request by yuliia-dronova**
- [DEMO LINK](https://yuliia-dronova.github.io/layout_stars/)
- [TEST REPORT LINK](https://yuliia-dronova.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <h1 hidden>Stars</h1>
```
Comment (VETALEK) [inline src/index.html:11]:
```suggestion
```

you don't need it (you even hid it) 😉

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```
Comment (VETALEK) [inline src/style.css:16]: the rightmost star don't need a `marign` 🥲

image

**Review by VETALEK — CHANGES_REQUESTED** [review]
Good start 👍

File: src/index.html (hunk 1 above)
Comment (yuliia-dronova) [reply src/index.html:11]: Ok:)
But earlier Misha (as far as I remember) said that it is essential to keep structure as it is has to be. And unrelevant elements can be just hidden.. Is it true?)

Comment (VETALEK) [reply src/index.html]: you may hide element while developing, but you don't need element that user can't even see in production 😉

Comment (VETALEK) [reply src/index.html]: it is just useless 🙃

**Review by VETALEK — APPROVED** [review]
Fire 🔥

File: src/index.html (hunk 1 above)
Comment (yuliia-dronova) [reply src/index.html]: Clear, thank you ^^)

**Pull request closed without merging**